src/styles/global.scss

```
@@ -6,3 +6,10 @@
   font-weight: normal;
   font-style: normal;
 }
+
+* {
```

there is no need to set these properties on every element, you can just set them on `body` and they will be inherited in the whole document.

```suggestion
body {
```